multi-tenancy-observability/README.md

```diff
@@ -0,0 +1,154 @@
+---
+creation_date: 2024-02-08
+issues:
+- https://github.com/giantswarm/roadmap/issues/2771
+owners:
+- https://github.com/orgs/giantswarm/teams/team-atlas
+state: review
+summary: How Giant Swarm manages multi-tenancy to isolate observability data per tenants (i.e. metrics, logs, traces).
+---
+
+# Multi-tenancy for observability
+
+## Glossary
+
+- `tenant`: a group of users who share a common access with specific privileges to observability data
+- `observability data`: all data related to metrics, logs, traces, profiles
+- `read path`: path used by users to access observability data
+- `write path`: path used by components to store observability data
+
+## Introduction
+
+Giant Swarm managed components produce a lot of observability data (metrics, logs, traces and so on).
+For instance, we have reached our ingestion limits in Prometheus more than a year ago.
```

Comment:
Please use absolute terms ("a year ago" is undefined if somebody reads this later)